layers/GraphQLAPIForWP/plugins/graphql-api-for-wp/graphql-api.php

```diff
@@ -15,40 +15,36 @@
 GitHub Plugin URI: GraphQLAPI/graphql-api-for-wp-dist
 */
 
+use GraphQLAPI\GraphQLAPI\Plugin;
+use GraphQLAPI\GraphQLAPI\PluginInfo;
+
 // Exit if accessed directly
 if (!defined('ABSPATH')) {
     exit;
 }
 
-use GraphQLAPI\GraphQLAPI\Plugin;
-
 /**
  * Make sure this plugin is not duplicated.
  * For instance, if zip file already exists in Downloads folder, then
  * the newly downloaded file will be renamed (eg: graphql-api(2).zip)
  * and the plugin will exist twice, as graphql-api/... and graphql-api2/...
  */
-if (defined('GRAPHQL_API_VERSION')) {
+if (class_exists('\GraphQLAPI\GraphQLAPI\PluginInfo')) {
```

Comment:
@leoloso PHP 5 has `::class` - so `class_exists(PluginInfo::class)`

Reply:
But if class PluginInfo doesn't exist, then `PluginInfo::class` will throw an error! That's why I need to pass it as a string, so it never fails

Reply:
Are you sure? https://3v4l.org/tdg9H

Reply:
Interesting, it seems you're right! OK, I'll change that in my next PR